manager/assets/modext/util/multiuploaddialog.js

```diff
@@ -166,6 +166,10 @@
                         if (upload.progress) {
                             upload.progress.hide();
                         }
+                        // Reload the modx-browser tree/dataview
+                        if (typeof browser.run !== "undefined") {
```

Comment:
Could there be a case where `browser` itself is null?

Reply:
@Mark-H It shouldn't happen, but yes, it possibly could happen :). I will add a check for `browser`.